<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	font-size: 14px;
}

QPushButton {
	font-style: inherit;
	font-size: 24px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="styleSheet">
       <string notr="true">QSplitter::handle:horizontal {
	background-color: #aaa;
	border: 2 dashed gray;
}</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="dataWidget" native="true">
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="group">
          <property name="title">
           <string>Данные</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QTableWidget" name="dataTable">
             <property name="columnCount">
              <number>5</number>
             </property>
             <column>
              <property name="text">
               <string>ФИО</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Дата рождения</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Новый столбец</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Телефон</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Адрес</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="controlWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupRemove">
          <property name="title">
           <string>Удалить выделенную запись</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxConfirmation">
             <property name="text">
              <string>Подтверждать удаление</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonRemove">
             <property name="text">
              <string>Удалить</string>
             </property>
             <property name="icon">
              <iconset resource="src/resources.qrc">
               <normaloff>:/images/icons/BD_Drop.png</normaloff>:/images/icons/BD_Drop.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupLoadSave">
          <property name="title">
           <string>Загрузить/Сохранить базу</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QRadioButton" name="radioButtonBin">
             <property name="text">
              <string>.bin</string>
             </property>
             <property name="icon">
              <iconset resource="src/resources.qrc">
               <normaloff>:/images/icons/BD_Bin.png</normaloff>:/images/icons/BD_Bin.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="buttonSave">
             <property name="text">
              <string>Сохранить</string>
             </property>
             <property name="icon">
              <iconset resource="src/resources.qrc">
               <normaloff>:/images/icons/BD_Save.png</normaloff>:/images/icons/BD_Save.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="buttonLoad">
             <property name="text">
              <string>Загрузить</string>
             </property>
             <property name="icon">
              <iconset resource="src/resources.qrc">
               <normaloff>:/images/icons/BD_Load.png</normaloff>:/images/icons/BD_Load.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="autoRepeatDelay">
              <number>300</number>
             </property>
             <property name="autoRepeatInterval">
              <number>100</number>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioButtonTxt">
             <property name="text">
              <string>.txt</string>
             </property>
             <property name="icon">
              <iconset resource="src/resources.qrc">
               <normaloff>:/images/icons/BD_Txt.png</normaloff>:/images/icons/BD_Txt.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupAdd">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Добавление записи</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="horizontalSpacing">
            <number>9</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelAddres">
             <property name="text">
              <string>ФИО</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditFio"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelBornDate">
             <property name="text">
              <string>Дата рождения</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditBornDate">
             <property name="inputMask">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelGroup">
             <property name="text">
              <string>Группа</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditGroup"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelNumber">
             <property name="text">
              <string>Телефон</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditNumber">
             <property name="inputMask">
              <string>999-99-99</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelAddres_2">
             <property name="text">
              <string>Адрес</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditAddres"/>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QPushButton" name="buttonAdd">
             <property name="text">
              <string>Добавить</string>
             </property>
             <property name="icon">
              <iconset resource="src/resources.qrc">
               <normaloff>:/images/icons/BD_Add.png</normaloff>:/images/icons/BD_Add.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src/resources.qrc"/>
 </resources>
 <connections/>
</ui>
